common/models/user.js

```diff
@@ -756,10 +756,20 @@ module.exports = function(User) {
 
     UserModel.validatesFormatOf('email', { with: re, message: g.f('Must provide a valid email') });
 
-    // FIXME: We need to add support for uniqueness of composite keys in juggler
```

Comment:
Is this FIXME has been taken care of in juggler? If not, I would leave it there in the code as a reminder.